File: codegen/codegen-erlang/src/test/resources/ir/client_dispatch_get_name_sigv4.expected.erl
Req = http_service_rest_json_1:encode_get_name_request(Input),
SignedReq = case maps:get(credentials, Config, undefined) of
    undefined -> Req;
    _ -> http_service_sigv4:sign(Config, get_name, Req)
end,
case runtime_http:dispatch(Config, SignedReq) of
    {ok, Resp} ->
        http_service_rest_json_1:decode_get_name_response(Resp);
    {error, Reason} ->
        {error, Reason}
end.

File: codegen/codegen-elixir/src/test/resources/ir/client_dispatch_get_name_sigv4.expected.ex
req = HttpServiceRestJson1.encode_get_name_request(input)
signed_req = case Map.get(config, :credentials) do
  nil -> req
  _ -> HttpServiceSigv4.sign(config, :get_name, req)
end
case RuntimeHttp.dispatch(config, signed_req) do
  {:ok, resp} -> HttpServiceRestJson1.decode_get_name_response(resp)
  {:error, reason} -> {:error, reason}
end
